requirements/edx/github.txt

```diff
@@ -67,7 +67,7 @@ git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 
 # Used for testing
 git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
-git+https://github.com/edx/pa11ycrawler.git@a963bbb7c4f861b24b5f66bfc5259d3fa207cca8#egg=pa11ycrawler
+git+https://github.com/edx/pa11ycrawler.git@a963bbb7c4f861b24b5f66bfc5259d3fa207cca8#egg=pa11ycrawler==0.1a
```

Comment:
Ugh, I'm looking now and see in the setup.py that the version is 0.1, but [according to GitHub the current release](https://github.com/edx/pa11ycrawler/releases) is 0.0.2. 

I agree that we should take care of that soon, as it's confusing and could waste a lot of somebody's time by misleading them.